{{cookiecutter.repo_name}}/docs/requirements.yaml

```
@@ -1,5 +1,10 @@
 name: {{cookiecutter.repo_name}}-docs
 channels:
+{% if cookiecutter.dependency_source == 'Prefer default anaconda channel with pip fallback' %}
+  - base
```

```suggestion
  - defaults
```

Is this meant to be 'defaults' or something else?

---

Yeah sorry that's my bad, got muddled with something else.